processing/src/main/java/io/druid/query/groupby/GroupByQueryEngine.java

```diff
@@ -313,11 +311,11 @@ public RowIterator(GroupByQuery query, final Cursor cursor, ByteBuffer metricsBu
       dimensionSpecs = query.getDimensions();
       dimensions = Lists.newArrayListWithExpectedSize(dimensionSpecs.size());
       dimNames = Lists.newArrayListWithExpectedSize(dimensionSpecs.size());
-
       for (int i = 0; i < dimensionSpecs.size(); ++i) {
         final DimensionSpec dimSpec = dimensionSpecs.get(i);
+        final String dimension = dimSpec.getDimension();
```

Comment:
HA no they don't. Probably an artifact from some prior edits that were reverted.